### ISSUE TITLE
logsumexp was finally removed from scipy.misc and moved to scipy.special

### ISSUE THREAD

**Issue by dprada** [choderalab/yank#1161]
Hi!

The following line does not longer work:

```
from scipy.misc import logsumexp
```

It has to be replaced by:

```
from scipy.special import logsumexp
```

This was already reported in https://github.com/scipy/scipy/issues/9546.

**Comment by dprada** [comment]
This was fixed almost fixed already. The only file to be fixed is multistateanalyzer.py. A PR was submitted with the small modification.

**Comment by andrrizzi** [comment]
This was fixed in YANK in #1162. A PR for openmmtools has also been opened (choderalab/openmmtools#423).
